spec/models/product_price_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+LOCAL_PREFIX_PRODUCT_PRICE = "activerecord.errors.models.product_price.attributes"
+
+RSpec.describe ProductPrice, type: :model do
+  subject { build(:product_price) }
+  describe "validations" do
+    describe "price" do
+      it {
+        should validate_presence_of(:price)
+      }
+      it {
+        expect(subject.price).to be_a(Float)
+      }
+    end
+    describe "category" do
+      it {
+        should validate_presence_of(:category)
+      }
+      it {
+        should validate_inclusion_of(:category).in?([0, 1, 2])
+      }
+      describe "enum_names" do
+        it {
+          subject.category = 0
```

Comment:
use let to change category

Reply:
let cant be used in "it" block

![image](https://user-images.githubusercontent.com/93429979/185362786-b78970b5-9492-4658-b473-447c28f77f2c.png)

if let is out of "it" block, that dousnt change category number

![image](https://user-images.githubusercontent.com/93429979/185362973-766e43c6-780c-47fa-a2c4-5dc0cd002105.png)
(by default category is "MEDIUM")